### DISCUSSION TITLE
Вход в айти

### DISCUSSION THREAD

**Discussion by omgpiu** [frontendu/themes#7] [Темы]
Отучился за 8  месяцев на реакт-макаку, работаю уже около полугода. Какой список книг можете посоветовать че читать, сейчас в процессе прочтения you dont know js , что дальше?

**Comment by Anatolii-Ovcharuk** [comment]
То что Вы читаете, очень хорошая книга, но их несколько (Автор - Кайл Симпсон): 
1. Асинхронная обработка и оптимизация. 
2. ES6 и не только. 
3. Область видимости и замыкания. 
4. Познакомьтесь, JavaScript.
5. Типы и грамматические конструкции. 
6. Замыкания и объекты.

Могу посоветовать еще просмотреть статьи в интернете по алгоритмам сортировок для разнообразия (+ на собеседовании не помешает).

Ещё интересно книга Беляев С. А. - Разработка игр на языке JavaScript
